config/vufind/config.ini

```diff
@@ -1694,10 +1694,18 @@ limit   = 3
 phrase = false
 ; Offer expansions on terms as well as basic replacements
 expand  = true
-; Turning on 'simple' spell checking will improve performance,
-;  by ignoring the more complicated 'shingle' (mini phrases)
-;  based dictionary.
-simple = false
+; Set the list of spellcheck dictionaries used in a Solr query. The
+; available dictionaries are defined in solrconfig.xml.
+; To replicate the legacy 'simple' option, to improve performance by ignoring
```

Comment:
It might be just as well not to mention the simple option; I think it's more likely to be confusing than helpful.

It might make sense to add a new rule to [VuFind\Config\Upgrade::upgradeConfig()](https://github.com/vufind-org/vufind/blob/dev/module/VuFind/src/VuFind/Config/Upgrade.php#L534) to look for and replace the simple setting.

Reply:
I added a rule, see below.

I'm not sure about removing the explanation about simple, but will do that if you like.  Given how difficult it is to actually configure the spellchecker, your point about shingles may still be relevant, but I can't say.

Reply:
I wasn't suggesting you delete the whole comment, just remove the phrase "To replicate the legacy 'simple' option," -- and start with "To improve performance..." :-)

Reply:
Got it!  Done.